_includes/releases/v21.1/v21.1.17.md

```diff
@@ -0,0 +1,61 @@
+
+<h3 id="v21-1-17-command-line-changes">Command-line changes</h3>
+
+- The `cockroach debug tsdump` command now allows viewing timeseries data even in cases of node failure by allowing users to rerun the command with the import filename set to `"-"`. [#78057][#78057]
+- Fixed a bug where starting [`cockroach demo`](../v21.1/cockroach-demo.html) with the `--global` flag would not simulate latencies correctly when combined with the `--insecure` flag. [#78173][#78173]
+
+<h3 id="v21-1-17-db-console-changes">DB Console changes</h3>
+
+- The `_status/nodes` endpoint is now available to all users with the `VIEWACTIVITY` role option, not just `admin` users. In the DB Console, the **Nodes Overview** and **Node Reports** pages will now display unredacted information containing node hostnames and IP addresses for all users with the `VIEWACTIVITY` role option. [#78296][#78296]
+
+<h3 id="v21-1-17-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug where draining nodes in a cluster without shutting them down could stall foreground traffic in the cluster. [#77494][#77494]
+- Fixed a bug that caused errors when attempting to create table statistics with [`CREATE STATISTICS`](../v21.1/create-statistics.html) or `ANALYZE` for a table containing an index which indexed only [virtual computed columns](../v21.1/computed-columns.html). [#77566][#77566]
+- Added a limit of seven concurrent asynchronous consistency checks per store, with an upper timeout of one hour. This prevents abandoned consistency checks from building up in some circumstances, which could lead to increasing disk usage as they held onto Pebble snapshots. [#77612][#77612]
+- Fixed a bug where the **Details** page was not loading for statements whose app name contains `/` were not properly loading their **Details** pages. [#77946][#77946]
+- Fixed a memory leak in the Pebble block cache. [#78262][#78262]
+- Fixed a bug that caused internal errors when `COALESCE` and `IF` [expressions](../v21.1/scalar-expressions.html) had inner expressions with different types that could not be cast to a common type. [#78345][#78345]
+- Fixed a bug that caused errors when trying to evaluate queries with `NULL` values annotated as a tuple type, such as `NULL:::RECORD`. [#78638][#78638]
+- Fixed a bug that caused the optimizer to generate invalid query plans which could result in incorrect query results. The bug, which has been present since version 21.1.0, can appear if all of the following conditions are true: 1) the query contains a semi-join, such as queries in the form: `SELECT * FROM t1 WHERE EXISTS (SELECT * FROM t2 WHERE t1.a = t2.a);`, 2) the inner table has an index containing the equality column, like `t2.a` in the example query, 3) the index contains one or more columns that prefix the equality column, and 4) the prefix columns are `NOT NULL` and are constrained to a set of constant values via a `CHECK` constraint or an `IN` condition in the filter. [#78976][#78976]
```

Comment:
```suggestion
- Fixed a bug that caused the [optimizer](../v21.1/cost-based-optimizer.html) to generate invalid query plans which could result in incorrect query results. The bug, which has been present since version v21.1.0, can appear if all of the following conditions are true: 1) the query contains a semi-join, such as queries in the form: `SELECT * FROM t1 WHERE EXISTS (SELECT * FROM t2 WHERE t1.a = t2.a);`, 2) the inner table has an index containing the equality column, like `t2.a` in the example query, 3) the index contains one or more columns that prefix the equality column, and 4) the prefix columns are `NOT NULL` and are constrained to a set of constant values via a `CHECK` constraint or an `IN` condition in the filter. [#78976][#78976]
```